source/core_mqtt_serializer.c

```diff
@@ -955,7 +955,7 @@ static MQTTStatus_t deserializeConnack( const MQTTPacketInfo_t * pConnack,
         if( ( pRemainingData[ 0 ] & MQTT_PACKET_CONNACK_SESSION_PRESENT_MASK )
             == MQTT_PACKET_CONNACK_SESSION_PRESENT_MASK )
         {
-            LogWarn( ( "CONNACK session present bit set." ) );
+            LogInfo( ( "CONNACK session present bit set." ) );
```

Comment:
Thanks. I'm pretty sure this was a leftover from v4_beta, where the session present flag wasn't given back to the application